doc/Core.xml

```diff
@@ -2948,6 +2951,15 @@ onvif://www.onvif.org/name/ARV-453
                   <entry>UserConfigNotSupported</entry>
                   <entry>Indicates no support for user configuration.</entry>
                 </row>
+                <row>
+                  <entry>
+                    <para>Addons</para>
+                  </entry>
+                  <entry>
+                    <para>Indicates the support for ONVIF Addons. See section TODO.</para>
```

Comment:
Not sure where to put the addon enum list yet in this doc...

Reply:
You can refer to the same section where you defined AddOn along with Profile.
Regarding including AddOn List, I am not sure whether we need to define here in core spec, because we don't list the Profiles anywhere in core spec, if required, you could add in the same section where AddOn is defined.

Reply:
I decided to remove the enum list.